src/sdkRuntimeModels.ts

```diff
@@ -217,14 +200,52 @@ export interface MParticleWebSDK {
         eventOptions?: SDKEventOptions
     ): void;
     logBaseEvent(event: BaseEvent, eventOptions?: SDKEventOptions): void;
+    logError(error: IErrorLogMessage, attrs?: SDKEventAttrs): void;
+    logLink(selector: string, eventName: string, eventType: valueof<typeof EventType>, eventInfo: SDKEventAttrs): void;
+    logForm(selector: string, eventName: string, eventType: valueof<typeof EventType>, eventInfo: SDKEventAttrs): void;
+    logPageView(eventName: string, attrs?: SDKEventAttrs, customFlags?: SDKEventCustomFlags, eventOptions?: SDKEventOptions): void;
+    setOptOut(isOptingOut: boolean): void;
     eCommerce: SDKECommerceAPI;
-    logLevel: string;
-    ProductActionType: SDKProductActionType;
+    isInitialized(): boolean;
+    ProductActionType: valueof<typeof ProductActionType>;
+    ready(f: Function): void;
+    reset(instance: IMParticleWebSDKInstance): void;
+    setAppName(name: string): void;
+    setAppVersion(version: string): void;
+    setOptOut(isOptingOut: boolean): void;
+
+    // https://go.mparticle.com/work/SQDSDKS-7063
+    startTrackingLocation(callback?: Callback): void;
+
+    stopTrackingLocation(): void;
     generateHash(value: string): string;
+    setIntegrationAttribute(integrationModuleId: number, attrs: IntegrationAttribute): void;
+    getIntegrationAttributes(integrationModuleId: number): IntegrationAttribute;
+}
+
+// https://go.mparticle.com/work/SQDSDKS-4805
```

Comment:
not sure if this comment needs to be here.  not a big deal and can be removed in the next PR